tutorials/GradeHelper.java

```diff
@@ -0,0 +1,46 @@
+public class GradeHelper {
```

Comment:
I don't think you need to include your repl.it code in your iP repository (unless you plan to integrate these functionalities in your iP, in which case it's probably fine).